### ISSUE TITLE
啥都没报就奇怪的停止了（qq：1003482256）

### ISSUE THREAD

**Issue by liboshi33** [PaddlePaddle/PaddleDetection#1421]
(paddle_huanjing) Y:\PaddleDetection>python -u tools/train.py -c configs/faster_rcnn_resnet101_fpn_2x_diban.yml
2020-09-16 09:36:26,500-WARNING: paddle.fluid.layers.matrix_nms OP not found, maybe a newer version of paddle is required.
2020-09-16 09:36:27,030-WARNING: Y:\PaddleDetection\ppdet\modeling\backbones\fpn.py:108
The behavior of expression A + B has been unified with elementwise_add(X, Y, axis=-1) from Paddle 2.0. If your code works well in the older versions but crashes in this version, try to use elementwise_add(X, Y, axis=0) instead of A + B. This transitional warning will be dropped in the future.
2020-09-16 09:36:27,036-WARNING: Y:\PaddleDetection\ppdet\modeling\backbones\fpn.py:108
The behavior of expression A + B has been unified with elementwise_add(X, Y, axis=-1) from Paddle 2.0. If your code works well in the older versions but crashes in this version, try to use elementwise_add(X, Y, axis=0) instead of A + B. This transitional warning will be dropped in the future.
2020-09-16 09:36:27,041-WARNING: Y:\PaddleDetection\ppdet\modeling\backbones\fpn.py:108
The behavior of expression A + B has been unified with elementwise_add(X, Y, axis=-1) from Paddle 2.0. If your code works well in the older versions but crashes in this version, try to use elementwise_add(X, Y, axis=0) instead of A + B. This transitional warning will be dropped in the future.
2020-09-16 09:36:27,434-INFO: If regularizer of a Parameter has been set by 'fluid.ParamAttr' or 'fluid.WeightNormParamAttr' already. The Regularization[L2Decay, regularization_coeff=0.000100] in Optimizer will not take effect, and it will only be applied to other Parameters!
W0916 09:36:27.707644  5952 device_context.cc:265] Please NOTE: device: 0, CUDA Capability: 61, Driver API Version: 10.2, Runtime API Version: 10.0
W0916 09:36:27.715622  5952 device_context.cc:273] device: 0, cuDNN Version: 7.6.
2020-09-16 09:36:29,732-WARNING: C:\Users\Administrator/.cache/paddle/weights\ResNet101_pretrained.pdparams not found, try to load model file saved with [ save_params, save_persistables, save_vars ]
2020-09-16 09:36:30,274-WARNING: C:\Users\Administrator/.cache/paddle/weights\ResNet101_pretrained.pdparams not found, try to load model file saved with [ save_params, save_persistables, save_vars ]
2020-09-16 09:36:30,287-WARNING: variable file [ C:/Users/Administrator/.cache/paddle/weights/ResNet101_pretrained/fc_0.w_0 C:/Users/Administrator/.cache/paddle/weights/ResNet101_pretrained/fc_0.b_0 ] not used
2020-09-16 09:36:31,067-INFO: places would be ommited when DataLoader is not iterable
W0916 09:36:31.088600  5952 build_strategy.cc:170] fusion_group is not enabled for Windows/MacOS now, and only effective when running with CUDA GPU.
2020-09-16 09:36:33,033-INFO: iter: 0, lr: 0.000125, 'loss_cls': '0.638281', 'loss_bbox': '0.010214', 'loss_rpn_cls': '0.695912', 'loss_rpn_bbox': '0.010581', 'loss': '1.354988', time: 0.000, eta: 0:00:00
2020-09-16 09:36:54,329-INFO: iter: 20, lr: 0.000170, 'loss_cls': '0.163843', 'loss_bbox': '0.013752', 'loss_rpn_cls': '0.691670', 'loss_rpn_bbox': '0.013607', 'loss': '0.878612', time: 1.106, eta: 3:07:43
2020-09-16 09:37:16,064-INFO: iter: 40, lr: 0.000215, 'loss_cls': '0.081523', 'loss_bbox': '0.000200', 'loss_rpn_cls': '0.613412', 'loss_rpn_bbox': '0.009793', 'loss': '0.717750', time: 1.090, eta: 3:04:34
2020-09-16 09:37:37,377-INFO: iter: 60, lr: 0.000260, 'loss_cls': '0.081587', 'loss_bbox': '0.000164', 'loss_rpn_cls': '0.185636', 'loss_rpn_bbox': '0.014967', 'loss': '0.307023', time: 1.066, eta: 3:00:06

**Comment by willthefrog** [comment]
代码或者配置有什么改动吗？ 版本信息？ 

也可以看下主机内存是不是溢出了?

**Comment by liboshi33** [comment]
> 代碼或者配置有什麼篩選嗎？版本信息？
> 
> 也可以看下主機內存是不是溢出了嗎？
没有啊，我就加了一个dataset信息就没别的了，内存应该不会，我16g的内存

**Comment by willthefrog** [comment]
跑原版dataset也是这样的吗？

**Comment by Joeygrant** [comment]
也发现了这个情况，突然就退出了，啥也没报
